packages/waas/tests/intents.spec.ts

```diff
@@ -127,11 +127,10 @@ describe('Payloads', () => {
             data: '0x223344'
           },
           {
-            type: 'delayedEncode',
+            type: 'contractCall',
```

Comment:
using the name "contractCall" for now, since this is inside of a "sendTransaction" above it, but, usually the "call" is related to read-only operations.

We could call it just "contract" or "contractTx" or "contractSend" .. if anyone has any other ideas, please let us know